src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -611,19 +645,25 @@ void MenuGame::initInput()
 
     game.addInput(
         Config::getTriggerExit(),
-        [this](ssvu::FT mFT) {
-            if(state != States::MOpts)
-            {
-                exitTimer += mFT;
-            }
+        [this](ssvu::FT mFT)
+        {
+            if(noActions) return;
+            if(state != States::MOpts) exitTimer += mFT;
```

Comment:
Ditto, please do not alter existing code style. Not a big deal, but it's just going to be reverted...